geo-types/src/lib.rs

```diff
@@ -1,3 +1,11 @@
+#![doc(html_logo_url = "https://avatars1.githubusercontent.com/u/10320338?s=1000&v=4")]
```

Comment:
Would it also work to use this url? https://raw.githubusercontent.com/georust/meta/master/logo/logo.png This might be better since we can swap it out at any point without breaking old docs

Reply:
👍